CHANGES.rst

```diff
@@ -1,7 +1,11 @@
 1.1.1 (unreleased)
 ==================
 
--
+jump
+----
+
+- Changes to flag both NIR snowballs and MIRI showers
+for  JP-#264. [#118]
```

Comment:
typo here: JP ticket number should be 2645

Reply:
fixed